workers/bubbleSort/run.py

```
@@ -0,0 +1,70 @@
+    resp = {
+        "code": MSG_TYPE["RESPONSE"],
+        "worker": True,
+        "type": "bubbleSort",
+        "value": sortedList,
+        "asp_uid": message["asp_uid"]
+    }
+    return resp
+
+async def handle_message(websocket, message):
+    message = json.loads(message)
+    asp_uid = message["asp_uid"]
```

Ruff (0.8.2)

Local variable `asp_uid` is assigned to but never used

Remove assignment to unused variable `asp_uid`

(F841)